### PR TITLE
Add autosave

### PR DESCRIPTION
**Pull request by t-eliks**

### CODE REVIEW

File: ToNote/Controls/NotePanel.cs [hunk 1]
```diff
@@ -36,16 +37,46 @@ public NotePanel()
...
             get => (Note)GetValue(NoteProperty);
             set => SetValue(NoteProperty, value);
         }
 
+        public static readonly DependencyProperty StatusProperty = DependencyProperty.Register("Status",
+           typeof(string), typeof(NotePanel), new FrameworkPropertyMetadata(null));
+
+        public string Status
+        {
+            get { return (string)GetValue(StatusProperty); }
```
Comment (Kodnot) [inline ToNote/Controls/NotePanel.cs]: Could be expression-bodied

File: ToNote/Controls/TodoControl.cs [hunk 2]
```diff
@@ -35,13 +35,22 @@ public TodoControl(Todo todo)
...
             };
         }
 
         public Todo Todo { get; private set; }
 
         public event RoutedEventHandler BackspacePressedWithAltShiftModifiers;
 
+        public event TextChangedEventHandler TextChanged;
+
+        public bool? Initializing { get { return extendedRTB?.Initializing; } }
```
Comment (Kodnot) [inline ToNote/Controls/TodoControl.cs]: Could be expression bodied, `public bool? Initializing => extendedRTB?.Initializing;`

File: ToNote/Models/Note.cs [hunk 3]
```diff
@@ -45,6 +45,22 @@ public string Description
...
         }
 
+        private string _Folder;
+
+        public string Folder
+        {
+            get => _Folder;
+            set
+            {
+                if (_Folder != value)
```
Comment (Kodnot) [inline ToNote/Models/Note.cs:55]: Could be inverted

**Review by Kodnot — APPROVED** [review]
A few minor nitpicks, otherwise looks good

**Comment by t-eliks** [comment]
[As a user, I want autosave for notes and todos so that I don't loose progress.](https://trello.com/c/vI90A5iS/26-as-a-user-i-want-autosave-for-notes-and-todos-so-that-i-dont-loose-progress)

**Pull request merged**